Branch-SDK/Branch-SDK/Branch+Validator.m

```diff
@@ -28,6 +28,11 @@ static inline void BNCAfterSecondsPerformBlockOnMainThread(NSTimeInterval second
     dispatch_after(BNCDispatchTimeFromSeconds(seconds), dispatch_get_main_queue(), block);
 }
 
+typedef struct {
+    NSString *uriScheme;
+    BOOL doesMatchServerScheme;
+} ValidatedClientUriSchemeModel;
```

Comment:
Prefix the struct definition with BNC.

Reply:
Also can't be a struct: use an object instead.

Structs can't hold obj-c objects.

Reply:
That should have triggered an error. :/